script/lint

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -x
+
+APP_DIRS="bigquery_etl sql_generators"
+TEST_DIR="tests"
+APP_AND_TEST_DIRS="$APP_DIRS $TEST_DIR"
+
+black --quiet --diff --check $APP_AND_TEST_DIRS
+flake8 $APP_AND_TEST_DIRS
+pydocstyle $APP_AND_TEST_DIRS
+mypy $APP_DIRS  --config-file="pyproject.toml"
```

Comment:
this is missing isort

Reply:
I believe we only use isort for pre-commit, i believe that's why i didn't include it here

Reply:
definitely not just for pre-commit, isort should be included with all our other linters

Reply:
I'm just saying that previously we only used it for pre-commit in this repository and I wanted to limit additional changes with this. can certainly be added but will require changing source files as a few don't pass

Reply:
ah, we had pytest-isort installed, but it was only being enforced via pre-commit, because it was missing from script/entrypoint. that's a separate bug, yeah.

Reply:
302 -> https://github.com/mozilla/bigquery-etl/pull/3384